test/Fail/DisallowedOverlaps.err

```diff
@@ -1,16 +1,22 @@
 Failed to solve the following constraints:
-  Resolve instance argument _18 : ⦃ a = a₁ : A ⦄ ⦃ b = b₁ : B ⦄ → A
+  Resolve instance argument (blocked on any())
```

Comment:
What does "blocked on any()" mean here?

Reply:
`any()` means it's stuck. Instance constraints really shouldn't have a separate blocker.